src/gengowatcher/main.py

```
@@ -7,207 +7,58 @@
-        or args.configure
-        or args.sync_session_from_browser
-        or args.check_session_from_browser
-    ):
+    if should_handle_lightweight_command(args):
         try:
             config = AppConfig()
             if handle_cli_config_commands(args, config, console):
                 sys.exit(0)
-        except Exception as e:
-            console.print(f"[error]Configuration error: {e}[/]")
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)